src/app/pages/StoryPage/index.jsx

```diff
@@ -62,6 +69,36 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
   const firstPublished = getFirstPublished(pageData);
   const lastPublished = getLastPublished(pageData);
   const aboutTags = getAboutTags(pageData);
+  const topStories = path(['additionalData', 'topStories'], pageData);
+  const featuresAndAnalysis = path(
+    ['additionalData', 'featuresAndAnalysis'],
+    pageData,
+  );
+  const topStoriesContent = topStories || secondaryColState.topStories;
+  const featuresAndAnalysisContent =
+    featuresAndAnalysis || secondaryColState.featuresAndAnalysis;
+
+  useEffect(() => {
+    const abortController = new AbortController();
+    const signal = abortController.signal;
+
+    const fetchAditionalData = async () => {
+      const data = await fetchPageData('/mundo/testdata', { signal: signal });
+      const { json } = data;
+      if (json) {
+        setSecondaryColState(json);
+      }
+    };
+
+    if (!topStories && !featuresAndAnalysis) {
+      console.log('fetching data');
+      fetchAditionalData();
+    }
+
+    return () => {
+      abortController.abort;
```

Comment:
would it be possible to have logic for fetching additional data for pages in like a top level HOC eg `withAdditionalPageData` ? not sure if it'd be possible but it would be interesting to have and could easily be reused.

Reply:
> shouldn't this be a function call ? [abort](https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort)

Thats a good spot. I have actually removed the the AbortController now as @simonsinclair pointed out it is not not supported in IE11

Reply:
> would it be possible to have logic for fetching additional data for pages in like a top level HOC eg `withAdditionalPageData` ? not sure if it'd be possible but it would be interesting to have and could easily be reused.

We could, although I imagine this part could be unique per page type, i.e. what data they fetch and what the store that data as in state. Would be worth making this more generic when he see another use case for the pattern.